examples/example-vite-kitchen-sink/src/components/starknet-provider.tsx

```
@@ -24,6 +28,13 @@ const cartridge = new CartridgeConnector({
     ],
 });
 
+let pa: PredeployedAccountsConnector[] = [];
+predeployedAccounts({
+    rpc: env.VITE_CONTROLLER_RPC,
+    id: "katana",
+    name: "Katana",
+}).then((p) => (pa = p));
```

Biome (1.9.4)

[error] The assignment should not be in an expression.

The use of assignments in expressions is confusing.
Expressions are often considered as side-effect free.

(lint/suspicious/noAssignInExpressions)